docs/src/ToricVarieties/AlgebraicCycles.md

```diff
@@ -7,9 +7,68 @@ Pages = ["AlgebraicCycles.md"]
 ```
 
 
-# Algebraic Cycles
+# The Chow ring and algebraic Cycles
 
-## Special attributes of toric varieties
+Algebraic cycles are elements of the Chow ring.
+We provide methods to work with such cycles.
+
+## Constructors
+
+### Generic constructors
+
+```@docs
+AlgebraicCycle(d::ToricDivisor)
+AlgebraicCycle(c::ToricDivisorClass)
+AlgebraicCycle(l::ToricLineBundle)
+AlgebraicCycle(cc::CohomologyClass)
+```
+
+### Addition, subtraction and scalar multiplication
+
+Addition of algebraic cycles `ac1` and `ac2` is implemented by
+`ac1+ac2`. Similarly, we can subtract the cycles by `ac1-ac2`.
+Scalar multiplication with `c` (this could be an integer,
+fmpz or even fmpq number) is supported by `c*ac1`.
```

Comment:
This is just me and my style, so feel free to ignore me here; but I would move away from fake names for objects ("what is c???") and write something more along these lines (also below). Again, feel free to ignore or do something completely different; at some point we should go over everything and find a common style, but for now we can experiment...

```suggestion
Algebraic cycles can be added and subtracted via the usual `+` and `-`
operators. Moreover, multiplication by scalars from the left is supported
for scalars which are integers or of type `fmpz` or `fmpq`.
```
I've written here "from the left" based on your example snippet, of course adjust this if right scalar multiplication is also supported.

Reply:
That is good. Just need to adjust it for the entire documentation of the toric varieties (likely a lot). Let me look into this...